community/neo4j/src/test/java/upgrade/StoreUpgraderTest.java

```diff
@@ -146,9 +128,6 @@ public StoreUpgraderTest( String version )
     public static Collection<String> versions()
     {
         return Arrays.asList(
-                StandardV2_0.STORE_VERSION,
```

Comment:
Good point! I think there is value in keeping those tests and extend them to all relevant formats. Let't keep that outside of this PR though.